.github/workflows/pre-submit.yaml

```diff
@@ -13,17 +13,20 @@ jobs:
     runs-on: ubuntu-20.04
     steps:
     - name: Checkout code
-      uses: actions/checkout@v2
+      uses: actions/checkout@v3
       with:
         fetch-depth: 0
 
     - name: Set up Go
-      uses: actions/setup-go@v2
+      uses: actions/setup-go@v3
       with:
         go-version: 1.18
 
-    - name: Build and test
-      run: make manifests generate docker-build
+    - name: Run checks and unit tests
+      run: make test
+
+    - name: Test container build
+      run: make container-build
```

Comment:
This is not logically equivalent to previous code, can you explain the motivation ?

Reply:
We still have `make manifests generate docker-build` in the new code.
In container-build  there is `test`, which includes `manifests generate`, and afterwards we have `docker-build`.
The new thing is adding the creation of  the bundle directory (`bundle` target), and build of the bundle image before PR is merged for checking if building the bundle image was successful.